python/tvm/tir/stmt.py

```diff
@@ -536,6 +536,12 @@ class BufferRegion(Object, Scriptable):
     def __init__(self, buffer: Buffer, region: List[Range]):
         self.__init_handle_by_constructor__(_ffi_api.BufferRegion, buffer, region)  # type: ignore
 
+    def __eq__(self, other):
```

Comment:
Not sure if it's safe. I tried doing similar things in Relax operators, but it may influence the behavior of `StructualEqual`, see  https://github.com/apache/tvm/blob/8569b9fe632b9488978bede30e169d457aafd8c9/python/tvm/relax/expr.py#L152